tests/components/sharePanel-test.js

```diff
@@ -70,11 +69,76 @@ describe('SharePanel', function () {
         TestUtils.Simulate.click(twitter);
         TestUtils.Simulate.click(facebook);
         TestUtils.Simulate.click(googlePlus);
+        OO={};
```

Comment:
These need to get reset after the test.  Otherwise they will live for all the other tests.

Reply:
I am resting back to null as you explained. Thanks.

Reply:
Can you fix the spacing?  00 = {}

Reply:
Fixed indentation and spacing issue.

Reply:
@gramcha This one is still wrong.